tests/prepare/feature/data/epel-enable.fmf

```diff
@@ -0,0 +1,5 @@
+provision+:
+    image: centos:stream8
+prepare:
+    how: feature
+    epel: enabled
```

Comment:
The metadata do not define the `execute` step thus this is not recognized as a `tmt` plan. This is the reason why tests are failing

```
:: [ 15:22:24 ] :: [  BEGIN   ] :: Running 'tmt run plan --name epel-enable provision --how virtual prepare'
/var/tmp/tmt/run-002

No plans found.
```

Reply:
Added.